source/_integrations/metoffice.markdown

```diff
@@ -16,7 +16,7 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `metoffice` weather platform uses the Met Office's [DataPoint API](https://www.metoffice.gov.uk/datapoint) for weather data. You can get an API key by registering for a Met Office [account](https://register.metoffice.gov.uk/WaveRegistrationClient/public/register.do?service=datapoint). As their website is not as straightforward, after registration and verifying your account you can login [here](https://register.metoffice.gov.uk/MyAccountClient/account/view) to retrieve your API key.
+The `metoffice` weather platform uses the Met Office's [DataPoint API](https://www.metoffice.gov.uk/datapoint) for weather data. You can get an API key by registering for a Met Office [account](https://register.metoffice.gov.uk/WaveRegistrationClient/public/register.do?service=datapoint). Details of the registration process can be found [here](https://www.metoffice.gov.uk/services/data/datapoint/getting-started). As their website is not as straightforward, after registration and verifying your account you can login [here](https://register.metoffice.gov.uk/MyAccountClient/account/view) to retrieve your API key.
```

Comment:
```suggestion
The `metoffice` weather platform uses the Met Office's [DataPoint API](https://www.metoffice.gov.uk/datapoint) for weather data. 

## Getting started

1. To get started with the Met Office's DataPoint API, you need to get an API key.
  * Register for a [Met Office account](https://register.metoffice.gov.uk/WaveRegistrationClient/public/register.do?service=datapoint). 
  * Check the [details of the registration process](https://www.metoffice.gov.uk/services/data/datapoint/getting-started). 
1. After registration and verification of your account, [login](https://register.metoffice.gov.uk/MyAccountClient/account/view) to retrieve your API key.
```
This is actually procedural content. I think readability is better if procedural content is formatted in procedure style.

Reply:
I like the idea @c0ffeeca7 but when I looked at the preview, the indentation looks a bit off:
![image](https://github.com/home-assistant/home-assistant.io/assets/567982/569f6cc0-7979-4c5b-903d-34eda11e899f)

I have restyled it slightly differently and hopefully you think it looks better too:
![image](https://github.com/home-assistant/home-assistant.io/assets/567982/6da6ab2f-e682-497f-8ec8-175eeee004e2)